Rewrite HTTP security BBEs (#3791)
ballerina-platform/ballerina-distribution: examples/http-service-ssl-tls/http_service_ssl_tls.md

# HTTP service - SSL/TLS

> The `http:Listener` can be configured to communicate through HTTPS by providing a certificate file and a private key file. These configurations can be provided through the `securedSocket` field of the `http:ListenerConfiguration`. This field accepts an `http:ListenerSecureSocket` record, which provides the SSL-related configurations. Use this to secure the communication and data transfer between the server and the client.

Author: ```suggestion
The `http:Listener` can be configured to communicate through HTTPS by providing a certificate file and a private key file. The certificate and the key can be provided through the `secureSocket` field of the listener configuration. Use this to secure the communication and data transfer between the server and the client.
```

> An `http:Listener` can be configured to communicate through HTTPS. To secure an `http:Listener` using HTTPS, the listener needs to be configured with a certificate file and a private key file. These configurations can be provided through the `securedSocket` field of the `http:ListenerConfiguration`. This field accepts an `http:ListenerSecureSocket` record, which provides the SSL-related configurations. Use this to secure the communication and data transfer between the server and the client.

Author: ```suggestion
The `http:Listener` can be configured to communicate through HTTPS. To secure an `http:Listener` using HTTPS, the listener needs to be configured with a certificate file and a private key file. These configurations can be provided through the `securedSocket` field of the `http:ListenerConfiguration`. This field accepts an `http:ListenerSecureSocket` record, which provides the SSL-related configurations. Use this to secure the communication and data transfer between the server and the client.
```

> The `http:Listener` can be configured to communicate through HTTPS. To secure an `http:Listener` using HTTPS, the listener needs to be configured with a certificate file and a private key file. These configurations can be provided through the `securedSocket` field of the `http:ListenerConfiguration`. This field accepts an `http:ListenerSecureSocket` record, which provides the SSL-related configurations. Use this to secure the communication and data transfer between the server and the client.

Author: ```suggestion
The `http:Listener` can be configured to communicate through HTTPS by providing a certificate file and a private key file. These configurations can be provided through the `securedSocket` field of the `http:ListenerConfiguration`. This field accepts an `http:ListenerSecureSocket` record, which provides the SSL-related configurations. Use this to secure the communication and data transfer between the server and the client.
```
